pages/SignIn.js

```diff
@@ -0,0 +1,39 @@
+import BasePage from '../pages/BasePage';
+
+class SignIn extends BasePage {
+
+  //WebElements
+  get username(){ return $('[name="username"]') }
+  
+  get password() {return $('[name="password"]')}
+
+  get submitbtn() {return $('#submit')};
+
+  get text() {return $('span')};
+
+  get checkbox() {return $('#remember-me')};
+
+  get signUpForm() {return $('a[href="/bank/signup"]')}
+
+  get logo() {return $('a[href="/bank/home"]')}
+
+  get headerSignIn() {return $('strong')}
+    
+  async submit() {
+    await this.submitbtn.click();
+  }
+  
+  async remember() {
```

Comment:
This can be more descriptive: Choose something like `activateRememberMeOption()`. Longer method names, that are more explicit/descriptive are usually better than shorter, less descriptive ones. Anything that saves you "interpretation time" and avoids going to see method definition is a win in maintainability on the medium-long term.